pages/settings/settings.js

```diff
@@ -347,7 +347,18 @@ function onKeyMapChange (e) {
       keyMapSettings = {}
     }
 
+    var currentKeyMap = {...defaultKeyMap, ...keyMapSettings}
```

Comment:
There's a `userKeyMap` function (from defaultKeyMap.js) that will do this already.